sdk/core/azure-core/inc/azure/core/io/body_stream.hpp

```diff
@@ -166,46 +166,58 @@ namespace Azure { namespace IO {
   class FileBodyStream : public BodyStream {
   private:
     // immutable
-#if defined(AZ_PLATFORM_POSIX)
-    int m_fd;
-#elif defined(AZ_PLATFORM_WINDOWS)
-    HANDLE m_hFile;
-#endif
+    FILE* m_fileStream;
```

Comment:
Do we need to save this as member? We never use it.

Reply:
Sure, I can remove it.